coderadar-ui/package.json

```diff
@@ -3,7 +3,8 @@
   "version": "0.2.3",
   "scripts": {
     "ng": "node --max_old_space_size=5048 ./node_modules/@angular/cli/bin/ng",
-    "start": "node --max_old_space_size=5048 ./node_modules/@angular/cli/bin/ng serve --prod",
+    "start": "node --max_old_space_size=5048 ./node_modules/@angular/cli/bin/ng serve",
+    "prodserve": "node --max_old_space_size=5048 ./node_modules/@angular/cli/bin/ng serve --prod --port 4201",
```

Comment:
A separate "prod" command isn't a bad idea. However, running `npm prodserve` doesn't do anything for me (it just tells me the command doesn't exist). Also, why does it start on port 4201?

Reply:
I added the script so I could have both a production and a debug server running.
We can remove this for now and discuss a solution if necessary.